core/models.py

```
@@ -0,0 +1,28 @@
+
+    symbol = models.CharField(max_length=10, db_column="symbol")  # KIS 내부 식별자
+    stock_code = models.CharField(
+        max_length=10, db_column="stock_code", null=True, blank=True
+    )  # 실제 6자리 종목코드
+    time = models.DateTimeField(db_column="time")
+    price = models.DecimalField(max_digits=12, decimal_places=2, db_column="price")
+    volume = models.BigIntegerField(db_column="volume")
+
+    class Meta:
+        db_table = "stock_ticks"
+        ordering = ["-time"]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)